### ISSUE TITLE
Consider using scikit scalers in normalization module

### ISSUE THREAD

**Issue by mjgleason** [NatLabRockies/reVeal#22]
The `reVeal.normalization` module contains methods for scaling series of numeric values to the 0 to 1 scale. Currently, two methods are implemented: `calc_minmax` and `calc_percentile`. The former has an implementation in `scikit-learn` that we could use instead, if desired (https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MinMaxScaler.html#sklearn.preprocessing.MinMaxScaler). There are also a number of other scalers available in `scikit-learn` that we could wrap, if needed, within the `normalization` module. 

I don't believe there is an existing method for `calc_percentile` in `scikit-learn`, so that one we would need to leave as-is.
